docs/how-tos/working-with-postgresql-data-types.rst

```diff
@@ -0,0 +1,96 @@
+Working with PostgreSQL data types
+==================================
+
+You may be wondering if PostgREST allows the use of special literal values, such as ``now`` for timestamps, ``yes`` for booleans or time values including the time zone. The answer is yes to all of them, that is because PostgREST makes use of PostgreSQL text representation to work with data types.
+
+Timestamps
+----------
+
+As mentioned above, you can use the time zone to filter or send data if needed. Let's use this table as an example:
```

Comment:
This connects nicely with the previous paragraph but I think each type section should remain independent, in case reordering happens.
```suggestion
You can use the time zone to filter or send data if needed. Let's use this table as an example:
```